code-challenges.test.js

```diff
@@ -26,37 +26,171 @@ const numbersArray2 = [24, 27, 30, 33, 36]
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // Pseudo code:
+// Input: Given objects with numbers 
+// Output : the number inside the arguement is evenly divisible by 3 or not
+// Create a test with expect statements for each of the variables provided.
+// // Process : Create a function called evenThree that will call an object as an arguement, add the string `is divisible by`, use modulo operand for division and set 3 strictly equal to a remainder of 0.
+// Creat a test that will producted good failure in jest
 
-// a) Create a test with expect statements for each of the variables provided.
-
-const object1 = { number: 15 }
+// const object1 = { number: 15 }
 // Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
+// const object2 = { number: 0 }
 // Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
+// const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
 
+// Create a test in Jest seeing the test have good failure because the function is undefined
+
+describe(`evenThree`,  () => {
+  it(`should return a string indicating if the number is divisible by three or not`, () => {
+    expect(evenThree(object1)).toEqual("15 is divisible by three");
+    expect(evenThree(object2)).toEqual("0 is divisible by three");
+    expect(evenThree(object3)).toEqual("-7 is not divisible by three");
+  })
+})
+
+
+// Function returns undefined 
+
+//  Output :  FAIL  ./code-challenges.test.js
+//   evenThree
+//   ✕ if the number inside the object is evenly divisible by three (1 ms)
+
+// ● evenThree › if the number inside the object is evenly divisible by three
+
+//   ReferenceError: evenThree is not defined
+
+// class Divide {
+//   constructor(number) {
+//     this.number = number;
+//   }
+
+//   answer() {
+//     return this.number % 3 === 0;
+//   }
+// }
+
+// const object1 = new Divide(15);
+// const object2 = new Divide(0);
+// const object3 = new Divide(-7);
+
+// const evenThree = (obj) => {
+  
+// };
+
 // b) Create the function that makes the test pass.
 
+// Make the test pass by completing the function
+//  PASS  ./code-challenges.test.js
+//   evenThree
+
+class Divide {
+    constructor(number) {
+      this.number = number;
+    }
+  
+    answer() {
+      return this.number % 3 === 0;
+    }
+  }
```

Comment:
There is no need to make the class `Divide` at all, as it is just producing essentially the same object as the given variables. The evaluation done within `answer` can be done within the conditional in your function.